CODEOWNERS

```
@@ -18,6 +18,7 @@ packages/firebase_database/* @collinjackson @kroikie
 packages/firebase_dynamic_links/* @bparrishMines
 packages/firebase_messaging/* @collinjackson @kroikie
 packages/firebase_ml_vision/* @bparrishMines
+packages/firebase_mlkit_language/* @rishab2113
```

Maintainers for this repo are generally Flutter team members
```suggestion
packages/firebase_mlkit_language/* @collinjackson
```

---

Interesting. Would it be possible to stay on as a secondary codeowner?

like such -  ```packages/firebase_mlkit_language/* @collinjackson @rishab2113```